src/lib/api/project/index.ts

```diff
@@ -56,3 +57,44 @@ export async function deleteProjects(
     throw err;
   }
 }
+
+export async function updateProject(
+  requestManager: requestsManager,
+  orgId: string,
+  projectId: string,
+  newBranch: string,
+): Promise<string> {
+
+  getApiToken();
+  getSnykHost();
+  debug('Listing integrations for org: ' + orgId);
+
+  if (!orgId && !projectId) {
+    throw new Error(
+      `Missing required parameters. Please ensure you have set: orgId and projectId.
+      \nFor more information see: https://snyk.docs.apiary.io/reference/projects/individual-project/update-a-project`,
+    );
+  }
+
+  const body = {
+    branch: newBranch,
+  };
+
+  const url = `/org/${orgId.trim()}/project/${projectId.trim()}`
+  const res = await requestManager.request({
+  verb: 'put',
+  url: url,
+  body: JSON.stringify(body),
+  useRESTApi: false,
+  });
+
+  const statusCode = res.statusCode || res.status;
+  if (!statusCode || statusCode !== 200) {
+    throw new Error(
+      'Expected a 200 response, instead received: ' +
+        JSON.stringify({ data: res.data, status: statusCode }),
+    );
+  }
+  
+  return res.data.branch;
```

Comment:
Let's return the whole project here so we can use it as we need for logging etc.